mifospay/src/main/java/org/mifos/mobilewallet/mifospay/faq/ui/FAQActivity.java

```diff
@@ -74,30 +73,6 @@ public void initViews() {
      */
     public void initListeners() {
```

Comment:
If there is no content left in the function, we can remove the function too.

Reply:
removed it. Thanks!